ui/components/multichain/pages/send/components/address-book.tsx

```diff
@@ -125,7 +134,11 @@ export const SendPageAddressBook = () => {
             internalAccounts={internalAccounts}
             searchForContacts={searchForContacts}
             searchForRecents={searchForRecents}
-            selectRecipient={(address = '', name = '') => {
+            selectRecipient={async (address = '', name = '', chainId = '') => {
```

Comment:
Just clarifying that the address book entires are chain specific

Does that mean users have to re-add the same entries per chain?

Does that make sense given the EOA would be controlled by the same private key?